source/global.cc

```diff
@@ -100,6 +100,4 @@ namespace aspect
     }
   }
 
-  // Output parallel statistics [bool]
-  const bool output_parallel_statistics = false;
```

Comment:
is there an associated parameter that is being declared and read? If so, what should we do with it?